tools/run-tests.py

```diff
@@ -237,16 +237,22 @@ def run_jerry_debugger_tests(options):
 
     return ret_build | ret_test
 
+def update_test_args_with_snapshot_tool(jerry_path, test_args):
+    if "--snapshot-tool" in test_args:
+        snapshot_idx = test_args.index("--snapshot-tool")
+        test_args[snapshot_idx] = "--snapshot-tool=%s-snapshot" % (jerry_path)
```

Comment:
is this rewrite really needed? patching up the cli options this way is a bit creepy, and the same effect could be achieved by simply adding this "deduce snapshot tool path from engine path" logic to the run-test-suite.sh script. (explanation following there...)